plugins/ua_config_standard.c

```diff
@@ -5,12 +5,19 @@
 #include "ua_log_stdout.h"
 #include "ua_network_tcp.h"
 #include "ua_accesscontrol_default.h"
+#include "ua_types_generated.h"
+#include "ua_securitypolicy_none.h"
+#include "ua_types.h"
 
-/*******************************/
-/* Default Connection Settings */
-/*******************************/
+#define ANONYMOUS_POLICY "open62541-anonymous-policy"
+#define USERNAME_POLICY "open62541-username-policy"
 
-const UA_EXPORT UA_ConnectionConfig UA_ConnectionConfig_standard = {
+ /*******************************/
+ /* Default Connection Settings */
+ /*******************************/
+
+const UA_EXPORT UA_ConnectionConfig UA_ConnectionConfig_standard =
```

Comment:
I will leave it as is, for now and change it, once this PR is merged. It's just too much stuff otherwise.